src/cljs/org/broadinstitute/firecloud_ui/page/method_repo/method_config_importer.cljs

```diff
@@ -242,17 +246,29 @@
       :else
       [table/Table
        {:columns [{:header "Type" :starting-width 100}
-                  {:header "Item" :starting-width 450
-                   :sort-by (fn [m]
-                              [(m "namespace") (m "name") (int (m "snapshotId"))])
-                   :filter-by (fn [m]
-                                (clojure.string/join "/" (map m ["namespace" "name" "snapshotId"])))
+                  {:header "Namespace" :starting-width 160
+                   :sort-by (fn [m] [(clojure.string/lower-case (m "namespace"))])
                    :sort-initial :asc
+                   :filter-by (fn [m] (str (m "namespace")))
+                   :as-text (fn [m] (str (m "namespace")))
```

Comment:
Since these two are the same, you can do `:filter-by :text`.  You can also get rid of the `str` wrap.